americangut/per_sample.py

```diff
@@ -228,6 +230,40 @@ def taxa_summaries(opts, sample_ids):
     return results
 
 
+def check_sequence_counts(opts, sample_ids):
+    """Errors if the sequence counts post filtering are < 1000
```

Comment:
This description seems inaccurate. This gives count info if under rarefaction_depth, correct, not errors?